### PR TITLE
Release 0.1.4

### PR DESCRIPTION
**Pull request by meain**
### What does this PR change?

-

### Does the PR depend on any other PRs or Issues? If yes, please list them.

-

### Checklist

I confirm, that I have...

- [ ] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`
- [ ] Ran [`helm-docs`](https://github.com/norwoodj/helm-docs) to update docs for chart (if values.yaml file was changed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## [0.1.5] - 2022-09-30
+### Added
+- Add arm builds for images from [meain](https://github.com/meain)
```
Comment (akshay196) [inline CHANGELOG.md:9]: Is this change related to the helm-chart?

Comment (meain) [reply CHANGELOG.md:9]: Not really, but since this is a major change in deployment and since there is no other specific change, I thought this could go in here.

Comment (meain) [reply CHANGELOG.md]: Dropped the entry.

**Review by akshay196 — APPROVED** [review]
